DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -66,14 +75,24 @@ public static void Main(string[] args)
 
     private static void ConfigureHttpRequestPipeline(WebApplication app)
     {
+        // Ensure we do not lose X-Forwarded-* Headers when behind a Proxy
```

Comment:
This is existing code that has been rearranged. Is the ordering important? If so then the comment and/or commit message should reflect that so it doesn't get ordered differently in future